Makefile

```diff
@@ -166,6 +164,11 @@ copy-files:
 	install -m 644 srv/salt/ceph/functests/1node/tuned/off/*.sls $(DESTDIR)/srv/salt/ceph/functests/1node/tuned/off
 	install -d -m 755 $(DESTDIR)/srv/salt/ceph/functests/1node/symlink
 	install -m 644 srv/salt/ceph/functests/1node/symlink/*.sls $(DESTDIR)/srv/salt/ceph/functests/1node/symlink
+	# functests/3node
+	install -d -m 755 $(DESTDIR)/srv/salt/ceph/functests/3node
+	install -m 644 srv/salt/ceph/functests/3node/*.sls $(DESTDIR)/srv/salt/ceph/functests/3node
+	install -d -m 755 $(DESTDIR)/srv/salt/ceph/functests/3node/rebuild
+	install -m 644 srv/salt/ceph/functests/3node/rebuild/*.sls $(DESTDIR)/srv/salt/ceph/functests/3node/rebuild
```

Comment:
please change all instances of "3node" to "3nodes" here

Reply:
I've updated the commit